variables.yml

```diff
@@ -247,6 +247,80 @@ networks:
           mim_usd: '0xdD6296BD7515271F7E4b10C3A87A2f9863fECa97'
           movr_usd: '0x3f8BFbDc1e79777511c00Ad8591cef888C2113C1'
           usdt_usd: '0xF80DAd54AF79257D41c30014160349896ca5370a'
+  moonbeam:
+    rpc_url: https://rpc.api.moonbeam.network
+    wss_url: wss://wss.api.moonbeam.network
+    chain_id: 1284
+    hex_chain_id: '0x504'
+    node_directory: /var/lib/moonbeam-data
+    parachain_release_tag: v0.15.1
+    parachain_sha256sum: c08e5f55bfc1dc7e2eec47727f0e710bc112f64b04c96843874e2f22a9badcd6
+    chain_spec: moonbeam
+    block_explorer: https://moonscan.io/
+    binary_name: moonbeam
+    min_gas_price: 100
+    block_time: 12
+    gas_block: 15M
+    gas_tx: 12.995M
+    node:
+      cores: 8
+      ram: 16
+      hd: 500
+    democracy:
+      launch_period:
+        days: 7
+        blocks: 50400
+      vote_period:
+        days: 14
+        blocks: 100800
+      fast_vote_period:
+        days: 1
+        blocks: 7200
+      enact_period:
+        days: 1
+        blocks: 7200
+      cool_period:
+        days: 7
+        blocks: 50400
+      min_deposit: 400
+      max_votes: 100
+      min_preim_deposit: 0.04
+      max_proposals: 100
+    total_annual_inflation: 5
+    staking:
+      collator_bond_min: 100000
```

Comment:
same comments as I had previously... I think we need to be consistent throughout so if you prefer these to be collator instead of candidate, the variables for Moonriver and Moonbase should be updated here and where used throughout the code base as well

Reply:
Thanks! I think I got them all